### DISCUSSION TITLE
Multicolumn content

### DISCUSSION THREAD

**Discussion by sg08234** [templaza/astroid-framework#387] [Q&A]
What is the best way to create multicolumn content? Good old <div> or do you recommend another technique?

Thanks
Michael

**Comment by paurray** [comment]
Not an expert by any means but I use JCE

https://www.joomlacontenteditor.net/

You can see the result here. I think it is quite ok?

https://finalbug.tv/en/examples-video-work

**Comment by sg08234** [comment]
Thanks,

JCE ist my standard Joomla-editor since “ever”. Did you have to customize JCE anyhow für usage within Astroid or even more detailed: How did you create the Astroid-div-styles:

<div class="astroid-container">
    <div class="astroid-offcanvas d-none d-init" id="astroid-offcanvas">
   <div class="burger-menu-button active">
    <button aria-label="Off-Canvas Toggle" type="button" class="button close-offcanvas offcanvas-close-btn">
        <span class="box">
            <span class="inner"></span>
        </span>
    </button>
</div>   <div class="astroid-offcanvas-inner">

or where did you know from?

Thaks again - Michael
________________________________

Wir erheben Ihre Daten zum Zweck der Vertragsdurchführung sowie zur Erfüllung von vertraglichen und vorvertraglichen Pflichten und beruht auf Artikel 6 Abs. 1 b) DSGVO. Eine Weitergabe der Daten an Dritte erfolgt nur soweit zur Vertragserfüllung notwendig. Sie sind berechtigt, Auskunft der bei uns über Sie gespeicherten Daten zu beantragen sowie bei Unrichtigkeit der Daten die Berichtigung oder bei unzulässiger Datenspeicherung die Löschung der Daten zu fordern. Ihnen steht des Weiteren ein Beschwerderecht bei der Aufsichtsbehörde zu.

Von: Paul David Murray ***@***.***>
Gesendet: Dienstag, 5. September 2023 11:13
An: templaza/astroid-framework ***@***.***>
Cc: Rocholl, Michael ***@***.***>; Author ***@***.***>
Betreff: Re: [templaza/astroid-framework] Multicolumn content (Discussion #387)


Not an expert by any means but I use JCE

https://www.joomlacontenteditor.net/

You can see the result here. I think it is quite ok?

https://finalbug.tv/en/examples-video-work

**Comment by paurray** [comment]
To the best of my knowledge JCE works out of the box with no cavets. Install and switch out as standard Joomla editor. It is over a year ago but pretty sure I just followed the instructions here and that was it. Nothing more nothing less: https://www.joomlacontenteditor.net/support/documentation/columns/creating-and-editing-columns

Going here:
https://www.joomlacontenteditor.net/support/changelog/editor

I see:

And

“2.9.6 27 April 2021 Added

“Added support for the Astroid template framework for editor content styling.”

**Comment by sg08234** [comment]
Thanks!

It looks that in my installation the template.css is not correctly "connected" to JCE --> I will reinstall Astroid und JCE asap an let you kow.

Michael
